dotnet/src/SemanticKernel.UnitTests/Functions/OrderedAIServiceSelectorTests.cs

```diff
@@ -75,8 +75,8 @@ public void ItGetsAIServiceConfigurationForTextGenerationByServiceId()
         Kernel kernel = builder.Build();
 
         var promptConfig = new PromptTemplateConfig() { Template = "Hello AI" };
-        var executionSettings = new PromptExecutionSettings();
-        promptConfig.AddExecutionSettings(executionSettings, "service2");
+        promptConfig.AddExecutionSettings(new(), "service2");
+        var executionSettings = promptConfig.ExecutionSettings["service2"];
```

Comment:
Why make this change? It looks less efficient.

Reply:
As we internally clone the `ExecutionSettings` and set the `ServiceId` automatically, the original code fails the test because the comparison `Assert.Equivalent` tries to compare the settings without a `ServiceId` with the cloned Settings with an auto-set `ServiceId`.